scalr/data_source_scalr_module_version.go

```diff
@@ -46,7 +46,19 @@ func dataSourceModuleVersionRead(ctx context.Context, d *schema.ResourceData, me
 	var version string
 	if v, ok := d.GetOk("version"); ok {
 		version = v.(string)
-		mv, err = scalrClient.ModuleVersions.ReadBySemanticVersion(ctx, module.ID, version)
+		ml, err := scalrClient.ModuleVersions.List(ctx, scalr.ModuleVersionListOptions{Module: module.ID, Version: &version})
```

Comment:
This function returns only the first page.

Reply:
you are right, but it's the same as ReadBySemanticVersion, in the `List` case we filter by version and the max possible count of result items is two, and it always fits the page. There is no need iterate through the pages. If you look into the [ReadBySemanticVersion](https://github.com/Scalr/go-scalr/blob/f377861c40d6e1906f6980286b8a36f9d189f12a/module_version.go#L119) you will see the same.  The result fits into one page.